lib/suggest.js

```diff
@@ -90,8 +92,6 @@ function suggest(value) {
         continue
       }
 
-      before = value.slice(0, position)
-      after = value.slice(position + 1)
```

Comment:
Why does the current approach `slice` the word at the position of the `character` in an unrelated `group` string such as the keyboard group "qwertzuop"?